docs/Toy/toy/rewrites/setup_riscv_pass.py

```diff
@@ -26,11 +26,8 @@ def match_and_rewrite(self, op: ModuleOp, rewriter: PatternRewriter):
             ),
         )
         data_section = riscv.AssemblySectionOp(".data", Region(Block()))
-        text_section = riscv.AssemblySectionOp(
-            ".text", rewriter.move_region_contents_to_new_regions(op.regions[0])
```

Comment:
The ".text" assembler directive is now inserted before every function in xDSL so this one is not needed